curvefs/test/mds/fs_manager_test2.cpp

```diff
@@ -406,5 +404,53 @@ TEST_F(FsManagerTest2, createHybridFs) {
     EXPECT_EQ(FsStatus::INITED, resultInfo.status());
 }
 
+TEST_F(FsManagerTest2, createFsNameInvalid) {
```

Comment:
suggestion: you can extract fsname check procedure into a free function(neither a member function nor static member function), so, you can directly test fsname check procedure directly rather than writing long meaningless code.